Dockerfile

```diff
@@ -32,7 +32,7 @@ COPY Gemfile \
      Gemfile.lock ./
 COPY gems/ gems/
 
-
+RUN gem install -N debase-ruby_core_source --version 0.10.13
```

Comment:
I'm not sure that we need sources in the production container.

Reply:
@semyon-estrin, What sparked this addition? Gem dependencies should be in `Gemfile`. If it's only for testing/development, it can be wrapped in the appropriate group.

Reply:
Thanks, removed and moved to Gemfile.